cargo-shuttle/src/lib.rs

```diff
@@ -236,6 +236,8 @@ impl Shuttle {
 
             self.load_project(&mut project_args)?;
             self.project_create(&self.client()?, IDLE_MINUTES).await?;
+        } else {
+            println!("Run `cargo shuttle project start` to create project environment on Shuttle");
```

Comment:
Nice! Perhaps we should also have a message about running `cargo shuttle deploy` after this else block?

Reply:
Not after the else block but inside the else block, as `project_create` will call `wait_with_spinner` which already has the `cargo shuttle deploy` line printed.

Reply:
Maybe we can mark this as resolved. We've clarified how the `println` should be placed in relation to `wait_with_spinner`.